apps/src/templates/studioHomepages/JoinSection.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import color from '@cdo/apps/util/color';
+import i18n from "@cdo/locale";
+import ProgressButton from '@cdo/apps/templates/progress/ProgressButton';
+
+const styles = {
+  main: {
+    borderWidth: 1,
+    borderStyle: 'solid',
+    borderColor: color.charcoal,
+    height: 68,
+    width: 940,
+    backgroundColor: color.white,
+    marginTop: 20,
+    marginBottom: 20
+  },
+  mainDashed: {
+    borderWidth: 5,
+    borderStyle: 'dashed',
+    borderColor: color.border_gray,
+  },
+  heading: {
+    fontFamily: '"Gotham 4r", sans-serif',
+    fontSize: 16,
+    fontWeight: 'bold',
+    letterSpacing: -0.2,
```

Comment:
It was in the original toast specs from Mike - I think it just compresses the text a little.